scripts/zones/Temple_of_Uggalepih/npcs/qm8.lua

```diff
@@ -10,7 +10,7 @@ require("scripts/globals/npc_util")
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
-    if npcUtil.tradeHas(trade, {{ xi.item.LA_THEINE_CABBAGE, 12 }}) and not GetMobByID(ID.mob.HABETROT):isSpawned() and
+    if npcUtil.tradeHas(trade, {{ xi.items.LA_THEINE_CABBAGE, 12 }}) and not GetMobByID(ID.mob.HABETROT):isSpawned() and
```

Comment:
Kind of odd seeing this here? If this is just fixing an issue with your elshimo audit that is fine.

Reply:
Yeah realized I made a typo in a previous commit that was already merged. This was me fixing it.